### PR TITLE
SudeepupdatedBorrowBook.java

### PR DESCRIPTION
**Pull request by ghost**
hey Sudeep,
I have to make the all the changes, please review it if something left let me know

### CODE REVIEW

File: src/BorrowBookControl.java [hunk 1]
```diff
@@ -5,23 +5,23 @@
 	
 	private BorrowBookUI ui;
 	
-	private library L;
-	private member M;
+	private library libary; //variable name should be in camle case form and as well as meaningful(L to Libarary)
```
Comment (ghost) [inline src/BorrowBookControl.java]: variable name should be in camle case form and as well as meaningful(L to Libarary)

File: src/BorrowBookControl.java [hunk 2]
```diff
@@ -5,23 +5,23 @@
 	
 	private BorrowBookUI ui;
 	
-	private library L;
-	private member M;
+	private library libary; //variable name should be in camle case form and as well as meaningful(L to Libarary)
+	private member member; //variable name should be in camle case form and as well as meaningful(M to member)
```
Comment (ghost) [inline src/BorrowBookControl.java]: variable name should be in camle case form and as well as meaningful(M to member)

File: src/BorrowBookControl.java [hunk 3]
```diff
@@ -5,23 +5,23 @@
...
+	private library libary; //variable name should be in camle case form and as well as meaningful(L to Libarary)
+	private member member; //variable name should be in camle case form and as well as meaningful(M to member)
 	private enum CONTROL_STATE { INITIALISED, READY, RESTRICTED, SCANNING, IDENTIFIED, FINALISING, COMPLETED, CANCELLED };
 	private CONTROL_STATE state;
 	
-	private List<book> PENDING;
-	private List<loan> COMPLETED;
-	private book B;
+	private List<book> pending;//I have change the variable name from PENDING to pending 
+	private List<loan> completed;// again here i have made a change of variable name from COMPLETED to completed
```
Comment (ghost) [inline src/BorrowBookControl.java:14]: again here i have made a change of variable name from COMPLETED to completed

**Comment by sushant8991** [comment]
all good i am approving your code

**Pull request merged**